libpod/runtime.go

```diff
@@ -217,6 +217,14 @@ func NewRuntime(options ...RuntimeOption) (runtime *Runtime, err error) {
 		if _, err := os.Stat(configPath); err != nil {
 			foundConfig = false
 		}
+
+		// containers/image uses XDG_RUNTIME_DIR to locate the auth file.
+		// So make sure the env variable is set.
+		err = os.Setenv("XDG_RUNTIME_DIR", GetRootlessRuntimeDir())
```

Comment:
This seems like a limitation in containers/image that should be patched there.  Is this "FIXME: work around containers/image's broken XDG_RUNTIME_DIR implementation"?  Is there an upstream PR fixing their implementation?